tests/conftest.py

```diff
@@ -1108,7 +1102,10 @@ def factory(
             status_tmp = ""
 
         project = project or project_factory()
-        storageclass = storageclass or storageclass_factory(interface)
+        if interface == constants.CEPHBLOCKPOOL:
+            storageclass = helpers.default_storage_class(interface_type=interface)
+        elif interface == constants.CEPHFILESYSTEM:
+            storageclass = helpers.default_storage_class(interface_type=interface)
```

Comment:
statement in if and elif are same

Reply:
Suggestion:
```
storageclass = storageclass or helpers.default_storage_class(interface_type=interface)
```

Reply:
This one we already discussed and my answer is same as previous comment, its easy to bring in when required. Right now testing is done using default storageclass and dont want to skew the results.